internal/provider/data_source_fmc_access_control_policy.go

```diff
@@ -181,12 +373,35 @@ func (d *AccessControlPolicyDataSource) Read(ctx context.Context, req datasource
 		return
 	}
 
+	resCats, err := d.client.Get(config.getPath()+"/"+url.QueryEscape(config.Id.ValueString())+"/categories?expanded=true&offset=0&limit=1000", reqMods...)
+	if err != nil {
+		resp.Diagnostics.AddError("Client Error", fmt.Sprintf("Failed to retrieve object (GET), got error: %s, %s", err, res.String()))
+		return
+	}
+
+	resRules, err := d.client.Get(config.getPath()+"/"+url.QueryEscape(config.Id.ValueString())+"/accessrules?expanded=true&offset=0&limit=1000", reqMods...)
```

Comment:
Do we need to support pagination here or do we limit the # of rules to 1000 for now?

Reply:
I'll add a limit check to 1000 and a GH issue to support pagination. The soundness of the entire approach is not yet 100% confirmed for me.

Reply:
That's fine.

Reply:
Added :)